src/Xamarin.Android.Build.Tasks/Xamarin.Android.EmbeddedResource.targets

```diff
@@ -35,7 +35,7 @@ This file is used by all project types, including binding projects.
   <Target Name="_ResolveLibraryProjectImports"
       DependsOnTargets="$(CoreResolveReferencesDependsOn)"
       Inputs="$(ProjectAssetsFile);$(MSBuildProjectFullPath);@(_MonoAndroidReferencePath);@(_MonoAndroidReferenceDependencyPaths);@(AndroidAarLibrary);$(_AndroidBuildPropertiesCache)"
-      Outputs="$(_AndroidStampDirectory)_ResolveLibraryProjectImports.stamp">
+      Outputs="$(_AndroidStampDirectory)_ResolveLibraryProjectImports.stamp;$(_AndroidLibraryProjectImportsCache)">
```

Comment:
So, I'm hesitant to modify `Outputs` when it's already using a `*.stamp` file. Does this actually need to modify `Inputs` to fix a problem?

Looking at this as-is, this could happen:

1. Incremental build
2. `_ResolveLibraryProjectImports` runs, because something changed
3. We save `$(_AndroidLibraryProjectImportsCache)` with "save if changed" logic:

https://github.com/dotnet/android/blob/99ba8135fc47358702a5918757651d53d7b6fabd/src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs#L136

4. Timestamp on `$(_AndroidLibraryProjectImportsCache)` does not change

5. From now on `_ResolveLibraryProjectImports` will run on every build, no matter what. `$(_AndroidLibraryProjectImportsCache)` is older than `Inputs`.

Reply:
So in the log I saw the .cache file did not exist and the .stamp file did. This is because the `_CleanIntermediateIfNeeded` target was somehow only deleting some of the files. 
`_CleanMonoAndroidIntermediateDir` was called, but something happened (and did not error) which resulting in the cache file disappearing, but the stamp file remaining in place. 
I do wonder if we need `ContinueOnError="true"` on all the calls in the `_CleanMonoAndroidIntermediateDir` target. So that we just keep going

Reply:
Maybe the Nuget Hash change is enough in this case.